buildkite/jobs.go

```diff
@@ -87,6 +87,10 @@ func (js *JobsService) UnblockJob(org string, pipeline string, buildNumber strin
 		return nil, nil, err
```

Comment:
I'm looking at https://buildkite.com/docs/apis/rest-api/jobs#unblock-a-job and that endpoint doesn't accept any query parameters. So I think lines 85-87 are redundant. It was added 5 years ago and maybe by default. The bug report in #136 mentions that query parameters are added but get ignored so I think we should actually just remove these lines as well to clean it up properly.

The rest looks good. Technically, I think this might be a bug on the server side since the `fields` body property is documented as optional. But this works too 👍

Reply:
@jradtilbrook , I have removed the code block that appends the query parameters to the endpoint [here](https://github.com/buildkite/go-buildkite/commit/fe3c18117b6cd9cea668a34efd9b6e858332c21d ) as suggested.